### DISCUSSION TITLE
Dropdown button menu

### DISCUSSION THREAD

**Discussion by vibhuh** [tailwindlabs/tailwindcss#12974] [Help]
I want to have my menu in top of drop down button. Because, my button is positioned in the bottom of the screen. So, I want to list my menu on top of dropdown button. I don't know, how to do this with Tailwind CSS.

Please help.

**Comment by wongjn** [comment]
Consider providing some code that demonstrates what you have so far, such as in a [Tailwind Play](https://play.tailwindcss.com).

**Reply by vibhuh** [reply]
Thanks for your reply.

https://play.tailwindcss.com/vBXrJTcDYY

**Reply by wongjn** [reply]
You could consider reversing the order of the DOM elements: https://play.tailwindcss.com/13DFICxfSI
